packages/react/src/components/CTA/__stories__/cta.stories.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.bx--card {
+  margin-top: 1rem;
+  margin-left: -1rem;
+}
```

Comment:
I don't think these are needed? Removing them makes the story layout identical to the web components version.

Reply:
You're right 🤔 Weird... my fork was bit behind when I worked on it. Perhaps that's why I added them cause I see no reason now lol. Thanks!